packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -338,16 +339,16 @@ export async function getUpdatedTokenDeposit(
     return tx
   }
 
-  const { parentToChildMsg: l1ToL2Msg } =
+  const { parentToChildMsg } =
     await getParentToChildMessageDataFromParentTxHash({
       depositTxId: tx.txId,
       isEthDeposit: false,
       parentProvider: getProviderForChainId(tx.parentChainId),
       childProvider: getProviderForChainId(tx.childChainId)
     })
-  const _l1ToL2Msg = l1ToL2Msg as ParentToChildMessageReader
+  const _parentToChildMsg = parentToChildMsg as ParentToChildMessageReader
```

Comment:
i know this line is not introduced by you but the order of this and the if conditional below is not ideal because someone could have accidentally misread the condition to use `_parentToChildMsg` and refactored it but because of the casting, eslint won't show an error when the conditional is deleted.

the assignment should go below the if condition

can you move it on this PR please?